sw/host/tests/manuf/manuf_cp_ast_test_execution/src/main.rs

```diff
@@ -109,11 +116,20 @@ fn manuf_cp_ast_text_execution_write_otp(opts: &Opts, transport: &TransportWrapp
 }
 
 fn manuf_cp_ast_text_execution_read_otp(opts: &Opts, transport: &TransportWrapper) -> Result<()> {
+    // The target reset in `connect_riscv_jtag` will complete the OTP HW_CFG partition locking.
```

Comment:
The code is not locking the partition at the moment I think?

Reply:
Oops, yeah removed comment.